### PR TITLE
Switch back to manalize

### PR DESCRIPTION
**Pull request by chalasr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Manala
```
Comment (ogizanagi) [inline README.md]: Manalize

File: README.md [hunk 2]
```diff
@@ -10,13 +10,13 @@ Installation
 
 #### Download:
 ```sh
-$ curl -L https://github.com/manala/manala/releases/download/v0.1.1/manala.phar > /usr/local/bin/manala
+$ curl -L https://github.com/manala/manalize/releases/download/v0.1.1/manala.phar > /usr/local/bin/manala
```
Comment (ogizanagi) [inline README.md]: Can we rename the phars in the releases ?

Edit: seems doable :)

File: composer.json [hunk 3]
```diff
@@ -27,10 +27,10 @@
...
     },
     "autoload": {
-        "psr-4": { "Manala\\": "src/" }
+        "psr-4": { "Manala\\Manalize\\": "src/" }
     },
     "autoload-dev": {
-        "psr-4": { "Manala\\Tests\\": "tests/" }
+        "psr-4": { "Manala\\Manalize\\Tests\\": "tests/" }
     },
     "bin": ["bin/manala"]
```
Comment (ogizanagi) [inline composer.json]: `bin/manalize`

**Review by ogizanagi — CHANGES_REQUESTED** [review]
We should also rename `MANALA_DIR` to `MANALIZE_DIR`

File: README.md (hunk 2 above)
Comment (ogizanagi) [reply README.md]:
``` diff
- curl -L https://github.com/manala/manalize/releases/download/v0.1.1/manala.phar > /usr/local/bin/manala
+ curl -L https://github.com/manala/manalize/releases/download/v0.1.1/manalize.phar > /usr/local/bin/manalize
```

File: README.md [hunk 4]
```diff
@@ -10,13 +10,13 @@ Installation
...
 $ chmod +x /usr/local/bin/manala
```
Comment (ogizanagi) [inline README.md]:
``` diff
- chmod +x /usr/local/bin/manala
+ chmod +x /usr/local/bin/manalize
```

File: README.md [hunk 5]
```diff
@@ -10,13 +10,13 @@ Installation
...
 ```
 
 #### Manual build:
 ```sh
-$ git clone git@github.com:manala/manala
+$ git clone git@github.com:manala/manalize
 $ make build
 $ mv manala.phar /usr/local/bin/manala
```
Comment (ogizanagi) [inline README.md]:
``` diff
- mv manala.phar /usr/local/bin/manala
+ mv manalize.phar /usr/local/bin/manalize
```

File: README.md [hunk 6]
```diff
@@ -10,13 +10,13 @@ Installation
...
 $ chmod +x /usr/local/bin/manala
```
Comment (ogizanagi) [inline README.md]:
``` diff
- chmod +x /usr/local/bin/manala
+ chmod +x /usr/local/bin/manalize
```

**Review by ogizanagi — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (chalasr) [reply README.md]: Yeah it is :) good catch, this and the next ones should be ok now

**Review by ogizanagi — APPROVED** [review]

**Review by ogizanagi — CHANGES_REQUESTED** [review]
The `Makefile` needs to be updated as well for the phar.

**Comment by chalasr** [comment]
Good catch, the gitignore too. Done

**Review by ogizanagi — CHANGES_REQUESTED** [review]
Another one (sorry 😅 ): The `Diff` command help part.

**Comment by nervo** [comment]
@chalasr you still have the right permisisons to rename the repo itself ?

**Comment by chalasr** [comment]
@nervo Yeah I got them, will do this after merging and refresh on travis.
@ogizanagi Should be ok now

**Review by ogizanagi — APPROVED** [review]

**Comment by ogizanagi** [comment]
Ok. I think the last issues are all fixed now. 3..2..1..merged !

**Pull request merged**